glean-core/python/glean/glean.py

```diff
@@ -50,12 +55,16 @@ class Glean:
     # and saved between test runs.
     _ping_type_queue: Set["PingType"] = set()
 
+    # The ping uploader implementation
+    _ping_uploader: BaseUploader = HttpClientUploader()
+
     @classmethod
     def initialize(
         cls,
         configuration: Optional[Configuration] = None,
         application_id: Optional[str] = None,
         data_dir: Optional[Path] = None,
+        ping_uploader: Optional[PingUploader] = None,
```

Comment:
Let's be consistent with what [Kotlin does](https://github.com/mozilla/glean/blob/0b2f2ceea6171a3c6551345f0075bf51031a053a/glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt#L94) and add that as part of the configuration object.